From 5db5dc0aaa9800c196066c76078947d941ee45ad Mon Sep 17 00:00:00 2001
From: Dave Stevenson <dave.stevenson@raspberrypi.com>
Date: Thu, 19 Jan 2023 19:51:49 +0000
Subject: [PATCH] dtoverlay: Update vc4-kms-dpi-generic for changed
 mediabus formats

Several of the media bus formats aren't merged upstream yet,
therefore the values change when upstream does introduce
new formats.

As vc4-kms-dpi-generic references the media bus formats explicitly
by number, update it to match.

Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.com>
---
 arch/arm/boot/dts/overlays/vc4-kms-dpi-generic-overlay.dts | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

--- a/arch/arm/boot/dts/overlays/vc4-kms-dpi-generic-overlay.dts
+++ b/arch/arm/boot/dts/overlays/vc4-kms-dpi-generic-overlay.dts
@@ -65,10 +65,10 @@
 
 		rgb565 = <&panel_generic>, "bus-format:0=0x1017",
 			<&dpi_node_generic>, "pinctrl-0:0=",<&dpi_16bit_gpio0>;
-		rgb565-padhi = <&panel_generic>, "bus-format:0=0x1020",
+		rgb565-padhi = <&panel_generic>, "bus-format:0=0x1022",
 			<&dpi_node_generic>, "pinctrl-0:0=",<&dpi_16bit_cpadhi_gpio0>;
-		bgr666 = <&panel_generic>, "bus-format:0=0x101f";
-		bgr666-padhi = <&panel_generic>, "bus-format:0=0x101e",
+		bgr666 = <&panel_generic>, "bus-format:0=0x1023";
+		bgr666-padhi = <&panel_generic>, "bus-format:0=0x1024",
 			<&dpi_node_generic>, "pinctrl-0:0=",<&dpi_18bit_cpadhi_gpio0>;
 		rgb666-padhi = <&panel_generic>, "bus-format:0=0x1015",
 			<&dpi_node_generic>, "pinctrl-0:0=",<&dpi_18bit_cpadhi_gpio0>;
